app/models/food.rb

```diff
@@ -0,0 +1,4 @@
+class Food < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods, dependent: :destroy
```

Comment:
include has many through
`has_many :recipes, through: :food_recipes, dependent: :destroy`

Reply:
Ok. I did it. Thank you for your valuable review.